src/popup/components/Popup.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC, useState } from "react";
+import { Results } from "./Results";
+import { SearchBar } from "./SearchBar";
+
+export const Popup: FC = () => {
```

Comment:
So typescripters don't really follow a philosophy of explicitly specifying types for top level/exported values? I know this from other languages. I guess a reason would be that if I want to write and export a functional component, I can declare the export as such and if I accidentally implement something else, TS won't let me.

Reply:
Some do. And definitely for your library API. But this component is only used inside of this module, so in my opinion there is no reason to explicitly specify types.
It is much, much more common to be annoyed by having to change the type together with the implementation, than to be helped by the type in finding a bug. Especially because at the call site you (i.e. Typescript) will notice the mistake anyway!